apps/frontend/src/app/3_organisms/DappHeader/DappHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+
+import {
+  Dropdown,
+  Header,
+  Menu,
+  MenuItem,
+  NavMenuItem,
+  DropdownSize,
+} from '@sovryn/ui';
+
+import { ConnectWalletButton } from '../../2_molecules';
+import { useWalletConnect } from '../../../hooks';
+
+export const DappHeader: FC = () => {
+  const { connectWallet, disconnectWallet, wallets, pending } =
+    useWalletConnect();
+
+  return (
+    <Header
+      logo={<div className="w-[102px] h-[28px] bg-white rounded" />}
```

Comment:
Why you use just a placeholder instead of the logo?

Reply:
SovrynLogo component isn't created yet, so placeholder is fine for now, but let's use text that says "Sovryn" instead of the current white rectangle